pkg/apis/app/v1beta1/application.go

```diff
@@ -74,8 +92,10 @@ func (a *Application) Observables(scheme *runtime.Scheme, rsrc interface{}, rsrc
 	if a.Spec.Selector == nil || a.Spec.Selector.MatchLabels == nil {
 		return observables
 	}
+	log.Printf(" matching labels:  %v\n", a.Spec.Selector.MatchLabels)
```

Comment:
What's the convention for including `\n` here? Each statement is recorded in a separate log entry, I normally never need to include `\n`.

Reply:
My Bad used to c/c++ style printf()'s
Will cleanup